Makefile

```diff
@@ -0,0 +1,42 @@
+IMAGE_REPO ?= quay.io/nmstate
```

Comment:
Missing IMAGE_REGISTRY so we can push to local registry

```make
IMAGE_REGISTRY ?= quay.io
IMAGE_REPO ?= $(IMAGE_REGISTRY)/nmstate
```

Reply:
Done